content/guides/exporters/supported-exporters/Node.js/stackdriver-trace.md

```diff
@@ -1,41 +1,47 @@
 ---
-title: "Stackdriver (Trace)"
+title: "Stackdriver (Tracing)"
 date: 2018-07-22T23:47:14-07:00
 draft: false
 weight: 3
-class: "resized-logo"
 ---
 
 ![](/images/logo_gcp_vertical_rgb.png)
 
-{{% notice note %}}
-This guide makes use of Stackdriver for visualizing your data. For assistance setting up Stackdriver, [Click here](/codelabs/stackdriver) for a guided codelab.
-{{% /notice %}}
+- [Introduction](#introduction)
+- [Installing the exporter](#installing-the-exporter)
+- [Creating the exporter](#creating-the-exporter)
+- [Viewing your metrics](#viewing-your-metrics)
+- [Viewing your traces](#viewing-your-traces)
+- [References](#references)
 
+## Introduction
 Stackdriver Trace is a distributed tracing system that collects latency data from your applications and displays it in the Google Cloud Platform Console.
+
 You can track how requests propagate through your application and receive detailed near real-time performance insights.
 Stackdriver Trace automatically analyzes all of your application's traces to generate in-depth latency reports to surface performance degradations,
 and can capture traces from all of your VMs, containers, or Google App Engine projects.
 
-OpenCensus Node has support for this exporter available through package:
+Stackdriver Monitoring provides visibility into the performance, uptime, and overall health of cloud-powered applications.
```

Comment:
I think this page is only about Stackdriver Tracing?

Reply:
Oops, updating, thanks!